manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -89,26 +89,25 @@ spec:
               path: /readyz
               port: 8000
           securityContext:
-            privileged: true # always requires privilege to be useful (install node plugin, etc)
+            privileged: false
             runAsGroup: 0
             runAsUser: 0
             runAsNonRoot: false
             # Both ambient and sidecar repair mode require elevated node privileges to function.
-            # But we don't need _everything_ in `privileged`, so drop+readd capabilities based on feature.
-            # privileged is redundant with CAP_SYS_ADMIN
-            # since it's redundant, hardcode it to `true`, then manually drop ALL + readd granular
-            # capabilities we actually require
+            # But we don't need _everything_ in `privileged`, so explicitly set it to false and
+            # add capabilities based on feature.
             capabilities:
```

Comment:
Asking the same question in https://github.com/istio/istio/pull/53478/files#r1792582294 since it didn't seem to get directly answered -- why we do not `drop: ALL`? Can we still?

Reply:
Yeah - good call I missed that this go-around. We should `drop: ALL` as a matter of course

Reply:
Agree, updated